op-batcher/flags/flags.go

```diff
@@ -127,6 +127,20 @@ var (
 		Value:   2 * time.Minute,
 		EnvVars: prefixEnvVars("ACTIVE_SEQUENCER_CHECK_DURATION"),
 	}
+	CheckRecentTxsDepthFlag = &cli.IntFlag{
+		Name: "check-recent-txs-depth",
+		Usage: "Indicates how many blocks back the batcher should look during startup for a recent batch tx " +
+			"on L1. This should help avoid duplicate batcher txs.",
```

Comment:
```suggestion
			"on L1. This can speed up waiting for node sync.",
```
the node sync avoids the duplicate batcher txs, while the recent tx check speeds up that sync waiting.